lib/ui/src/components/sidebar/Menu.tsx

```diff
@@ -47,7 +47,17 @@ export const MenuButton = styled(Button)<MenuButtonProps>(({ highlighted, theme
   padding: 7,
   '&:focus': {
     background: theme.barBg,
-    boxShadow: `${theme.color.secondary} 0 0 0 1px inset`,
+    boxShadow: 'none',
+  },
+  '&:focus:before': {
```

Comment:
This solution wont be immediately obvious for future maintainers. Can you add a comment that describes what's going on here that links to this PR?

Reply:
Agreed; this approach of creating a before/after selector to make an additional border is really useful, so documenting it out will help future devs a lot!

Reply:
We use this approach exclusively on Fluent UI controls. Might be good to open up an issue to standardize on this approach across the repo.